### DISCUSSION TITLE
Welcome to digitalpalireader Discussions!

### DISCUSSION THREAD

**Discussion by parthopdas** [kitamstudios/digitalpalireader#294] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by thesunshade** [comment]
Hello!

Is this the right place to report problems with the DPR? I did a search for the word "kaḷārikā" and nothing was found. I tried with every single thing checked. It is in fact here, but the search turned up nothing. 
https://www.digitalpalireader.online/_dprhtml/index.html?loc=m.0.0.0.2.6.0.m&para=17
